service/resource/project/resource.go

```diff
@@ -196,25 +197,30 @@ func (r *Resource) ProcessCreateState(obj, createState interface{}) error {
 		r.logger.Log("debug", "creating projects in the Kubernetes cluster")
 
 		for _, p := range projectsToCreate {
+			ID, err := strconv.Atoi(p.ID)
+			if err != nil {
+				return microerror.Mask(err)
+			}
```

Comment:
We need this now because the ID of the deployment event is an `int`. We should make this a string. #later